CHANGELOG.md

```diff
@@ -66,6 +66,22 @@ Format:
 --->
 
 <!--- CueAddReleaseNotes --->
+## [1.3.0-rc.0] March 17, 2020
+[1.3.0-rc.0]: https://github.com/datawire/ambassador/compare/v1.2.2...v1.3.0-rc.0
+
+### Ambassador API Gateway + Ambassador Edge Stack
+
+Feature: `edgectl install` provides better feedback for clusters that are unreachable from the public Internet 
+Feature: `edgectl install` supports KIND clusters
```

Comment:
I would argue these are AES-specific, as `edgectl install` does not support installing OSS.